demo3/Demo.jsx

```diff
@@ -148,12 +149,12 @@ let Demo = React.createClass({
                           type="checkbox"
                           onChange={this.handleDone.bind(null, date)}
                           checked={isDone}
-                        />
+                          />
```

Comment:
Nah... I think it is my autoformatter. Do you think it should not be all aligned?